README.md

```diff
@@ -122,29 +116,75 @@ This requires setting up [binfmt_misc](https://docs.kernel.org/admin-guide/binfm
 
 On most distributions, you can install QEMU user static to set up binfmt_misc handlers. For example, on Debian, use the command `apt install qemu-user-static`.
 
+### SELinux
 
-## Customizing
-Building Garden Linux is based on a [feature system](features/README.md).
+If you intend to build targets with the `_selinux` feature, some additional requirements apply to the build machine.
 
-| Feature Type | Includes |
-|---|---|
-| Platforms | `ali`, `aws`, `azure`, `gcp`, `kvm`, `metal`, ... |
-| Features | `container host`, `virtual host`, ... |
-| Modifiers | `_slim`, `_readonly`, `_pxe`, `_iso`, ... |
-| Element | `cis`, `fedramp`, `gardener` |
 
-if you want to build manually choose:
-```
-build.sh  --features <Platform>,[<feature1>],[<featureX>],[_modifier1],[_modifierX] destination [version]
-```
+Building the `_selinux` feature will not work on machines running in SELinux enforcing mode. Ideally, you should build on a machine with SELinux disabled. However, if you want to build with SELinux in permissive mode, this can be achieved by running the build as root with the `--privileged` flag.
+
+i.e.: `sudo ./build --privileged ${target}`
 
-Additionally, please find some `build.sh` example calls in the `Makefile`.
+**SELinux build workaround for Podman:**
+> :warning: **Warning:** By default, Podman has SELinux enabled. This can lead to issues when building the `_selinux` feature, as described in [issue #16](https://github.com/gardenlinux/builder/issues/16).
```

Comment:
```suggestion
**SELinux build workaround for Podman Desktop:**
> :warning: **Warning:** By default, [Podman Desktop](https://podman-desktop.io) has SELinux enabled. This can lead to issues when building the `_selinux` feature, as described in [issue #16](https://github.com/gardenlinux/builder/issues/16).
```